Rubberduck.VBEEditor/SafeComWrappers/VB/Abstract/ICodeModule.cs

```diff
@@ -6,6 +6,10 @@ public interface ICodeModule : ISafeComWrapper, IEquatable<ICodeModule>
     {
         IVBE VBE { get; }
         IVBComponent Parent { get; }
+        /// <summary>
+        /// Returns the code pane associated with the cod module.
+        /// Accessing this property will open the code pane if it is not open already.
+        /// </summary>
         ICodePane CodePane { get; }
```

Comment:
Just a thought - instead of exposing the property straight, we could consider wrapping it in `TryGetOpenCodePane()` and `OpenCodePane()`. That would provide the benefit of separating the action of getting a code pane from opening a code pane, and require the caller to be explicit about what it wants to do with the code pane.

Reply:
Currently, the SCWs copy and extend the VBE API. Removing this property would destroy that.

Moreover, this would require each access to a code pane to make a lot of COM calls: validating whether a code pane is open requires iterating over the open code panes exposed via `VBE.CodePanes`. 

So, although this separates getting the code pane from opening it, I am not a huge fan of this idea.